tests/TestCompose_1dtube_cxx.Ubuntu1604.home/Dockerfile.1dtubecxx

```diff
@@ -17,4 +17,17 @@ RUN git clone https://github.com/precice/elastictube1d.git
 
 RUN mkdir -p Data/Output/Postproc
 WORKDIR /home/precice/elastictube1d/cxx
+<<<<<<< HEAD
+<<<<<<< HEAD:tests/Test_1dtube_cxx.Ubuntu1604.home/Dockerfile
```

Comment:
Merge conflict?

Reply:
I missed some files by accident when merging, should be resolved now.